pygamelib/actuators.py

```diff
@@ -331,11 +331,13 @@ class PathFinder(Behavioral):
         method is going to circle between the waypoints
         (when the last is visited, go back to the first)
     :type circle_waypoints: bool
+    :param path_finding_mechanism: 0 - BFS, 1 - AStar
+    :type path_finding_mechanism: int
 
     """
 
     def __init__(
-        self, game=None, actuated_object=None, circle_waypoints=True, parent=None
+        self, game=None, actuated_object=None, circle_waypoints=True, parent=None,path_finding_mechanism=0
```

Comment:
What do you think of changing `path_finding_mechanism` to `algorithm`?
Also, if we use numbers to identify the algorithm (and that's a good idea) we should use a constant in the `constants` module. The reason is both for eventual reusability and for ease of use for users.

I suggest to add 2 constants :

- `ALGO_BFS = 90000100`
- `ALGO_ASTAR = 90000101`

Reply:
Thanks for the feedback.  I ll make another commit fixing everything you said. About the constants, where should I place them?